### DISCUSSION TITLE
Creating multiple projects from a single PyAEDT instance!

### DISCUSSION THREAD

**Discussion by Abhi-SimuTech** [ansys/pyaedt#3816] [Q&A]
Hi All,

For the purpose of a extremely large geometry creation, we have to break-down the geometry and build the parts into several different projects (for speeding up the geometry creation process). The question is - is it possible to initiate a PyAEDT instance and create a HFSS project (define units, variables, and some other properties etc.), then copy everything (variables, units, others etc.) of this project to create another project, close the first project and open the new project, and so on (sequentially)? Do you see any issue in doing that in PyAEDT?

Regards
Abhi

**Comment by Samuelopez-ansys** [comment]
Hi @Abhi-SimuTech ,

Yes, it is possible.

The only limitation is controlling more than one AEDT session with PyAEDT.

Inside the same desktop, you can control multiple projects, designs etc…
